mxcube3/routes/qutils.py

```diff
@@ -29,6 +29,12 @@
 UNCOLLECTED = 0x0
 READY = 0
 
+TASK_COLLECTED = 4
+TASK_COLLECT_FAILED = 2
+TASK_COLLECT_WARNING = 3
+TASK_RUNNING = 1
+TASK_UNCOLLECTED = 0
```

Comment:
@meguiraun correct me if I am wrong, but aren't TASK_* constants the same as the ones above? Like, `TASK_COLLECTED == COLLECTED`. So we don't need both...

Also, `WARNING` = 3 but this doesn't compose well, it would be better as 0x10 (16) I think - like, we can have something both `COLLECTED` and `WARNING` maybe. With 3 we cannot distinguish.

By the way I noticed we maybe made a mistake with READY/UNCOLLECTED = 0... We can't know if we have SAMPLE_MOUNTED and UNCOLLECTED. Well, maybe it's implicit if nothing is set it's READY/UNCOLLECTED...